api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -100,6 +106,17 @@ pub trait Rpc {
 		affiliate_fees: Option<Affiliates<AccountId32>>,
 		dca_parameters: Option<DcaParameters>,
 	) -> RpcResult<SwapPayload>;
+
+	#[method(name = "mark_btc_transaction_as_tainted", aliases = ["broker_markBtcTransactionAsTainted"])]
+	async fn mark_btc_transaction_as_tainted(&self, tx_id: cf_chains::btc::Hash) -> RpcResult<()>;
+
+	#[method(name = "open_btc_deposit_channels", aliases = ["broker_openBtcDepositChannels"])]
+	async fn open_btc_deposit_channels(
```

Comment:
nit: this sounds a bit like a method for opening channels. Maybe `get_open_deposit_channels` ?

(and return a chain-variant enum as above)

Reply:
Yes, done.